pinot-spi/src/main/java/org/apache/pinot/spi/config/table/UpsertConfig.java

```diff
@@ -79,6 +79,9 @@ public enum ConsistencyMode {
   @JsonPropertyDescription("Whether to preload segments for fast upsert metadata recovery")
   private boolean _enablePreload;
 
+  @JsonPropertyDescription("If we are using deletion + compaction we need to enable this for data consistency")
```

Comment:
This only associates with `_deletedKeysTTL`, so suggest moving it next to `_deletedKeysTTL` and make the name and description more specific. Currently people might misunderstand this to be associated with `_deleteRecordColumn`

Reply:
Updated! Let me know if we can go for a better naming.

Reply:
Have you pushed the change?

Reply:
Yes i renamed it to `_enableDeletedKeysCompactionConsistency`. It's updated in UpsertConfig class. Pushing a change to update in `UpsertContext` as well.